File: forum-api/DataAccess/DataObjects/Comment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace forum_api.DataAccess.DataObjects
{
    public partial class Comment
    {
        public int Id { get; set; }
        public string User { get; set; } = null!;
        public DateTime? CreationDate { get; set; }
        public DateTime? ModificationDate { get; set; }
        public string Content { get; set; } = null!;
        public int TopicId { get; set; }

        [NotMapped]
        public virtual Topic? Topic { get; set; } = null!;
    }
}

File: forum-front/src/models/comment.ts
export class Comment{

    public id : number;
    public creationDate : Date;
    public modificationDate : Date;
    public user : string;
    public content : string;
    public topicId : number
    
    constructor(id : number, creationDate : Date, modificationDate : Date, user : string, content : string, topicId : number){
        this.id = id;
        this.creationDate = creationDate;
        this.modificationDate = modificationDate;
        this.user = user;
        this.content = content;
        this.topicId = topicId;
    }
}